code/modules/mob/living/brain/posibrain.dm

```diff
@@ -165,8 +166,18 @@ GLOBAL_VAR(posibrain_notify_cooldown)
 	brainmob.real_name = brainmob.name
 	brainmob.forceMove(src)
 	brainmob.container = src
-	if(autoping)
-		ping_ghosts("created", TRUE)
+	if(autoping && A)
+		notify_ghosts("A positronic brain has been created in \the [A.name].", source = src, action=NOTIFY_ATTACKORBIT, flashwindow = FALSE, ignore_key = POLL_IGNORE_POSIBRAIN)
+	GLOB.poi_list |= src
+	LAZYADD(GLOB.mob_spawners[initial(name)], src) //Yogs -- Adds positronic brains to Spawner Menu
+
+/obj/item/mmi/posibrain/Destroy()
+	GLOB.poi_list -= src
+	var/list/spawners = GLOB.mob_spawners[initial(name)]
+	LAZYREMOVE(spawners, src)
```

Comment:
Shouldn't this be done when the posi is posessed?

Reply:
U H H H H H H H H H S H H H H H H H H H H H H H